Plugin/Framework/Search/Request/AbstractHandler.php

```diff
@@ -149,16 +143,12 @@ public function handle(array &$requestData)
                 $this->parsePageNumber($requestData),
                 $this->parseLimit($requestData)
             );
-        } catch (CmpException $e) {
-            $this->logger->exception($e);
+        } catch (Exception $e) {
+            $this->exception($e);
```

Comment:
@olsi-qose we need to sanity-check this.

Reply:
Looks sane